examples/example_transfer_transaction.py

```diff
@@ -0,0 +1,35 @@
+from pactus.crypto import CryptoConfig
+from pactus.crypto.address import Address
+from pactus.crypto.bls.private_key import PrivateKey
+from pactus.transaction.transaction import Transaction
+from pactus.types.amount import Amount
+
+
+def main() -> None:
+    CryptoConfig.use_testnet()
+
+    lock_time = 1_735_096
+    memo = "This is a test transaction"
+    amount = Amount.from_string("1.5")
+    fee = Amount.from_string("0.01")
+    receiver = Address.from_string("tpc1zk4eztwv3fs6l9g776cfwkgujham79hrrwn82q5")
+
+    sender = Address.from_string("tpc1z4rlzvq8lv92cnp0k3mk5jfr00kjnz0qvklvn3u")
+    sec = PrivateKey.from_string(
+        "tsecret1pzf33h72n9phxzaty5urh5ss4m33vvdfawyvesl5jtlt9a00tnwkqyngm6z"
+    )
+
+    tx = Transaction.create_transfer_tx(lock_time, sender, receiver, amount, fee, memo)
+    tx.sign(sec)
+    # 0001b8791a0080ade2041a5468697320697320612074657374207472616e7
```

Comment:
print it

Reply:
And verify it...

Reply:
If we print it, we will have a linter error.